data.sql

```diff
@@ -88,6 +88,54 @@ WHERE name IN ('Angemon', 'Boarmon');
 
 COMMIT;
 
+-- INSERT VETS DATA
+BEGIN;
+
+ALTER SEQUENCE vets_id_seq RESTART WITH 1;
+
+INSERT INTO vets(name, age, date_of_graduation) VALUES('William Tatcher', 45, '2000-04-23');
+INSERT INTO vets(name, age, date_of_graduation) VALUES('Maisy Smith', 26, '2019-01-17');
+INSERT INTO vets(name, age, date_of_graduation) VALUES('Stephanie Mendez', 64, '1981-05-04');
+INSERT INTO vets(name, age, date_of_graduation) VALUES('Jack Harkness', 38, '2008-06-06');
```

Comment:
- [ ] The requirement is for Jun 6. Please kindly adjust date so that it matches requirement

Reply:
Hello @AnselemOdims could you be more specific? In line 99 I'm injecting June 6th, right?